action.yml

```diff
@@ -14,6 +14,7 @@ inputs:
   base_url:
     description: "(Optional) For self-hosted bitwarden instances provide your https://your.domain.com"
     required: false
+    default: ""
```

Comment:
“Added default value to base_url, since the docs specifies that all required: false inputs needs a default value.”

What documentation are you referring to?

Reply:
You are right, it's not required but there is a note: https://github.com/actions/toolkit/tree/main/packages/core#inputsoutputs
`If required set to be false, the input should have a default value in action.yml.`